examples/GLLinePlotItem.py

```diff
@@ -30,14 +30,14 @@
 w.addItem(gz)
 
 def fn(x, y):
-    return np.cos((x**2 + y**2)**0.5)
+    return np.cos(np.hypot(x, y))
 
 n = 51
 y = np.linspace(-10,10,n)
 x = np.linspace(-10,10,100)
 for i in range(n):
     yi = np.array([y[i]]*100)
```

Comment:
The following syntax looks weird: A single element of an ndarray is extracted, replicated as a Python list, then converted back to an ndarray.
```python
yi = np.array([y[i]]*100)
```

Maybe numpy broadcasting could be used:
```python
yi = y[i]
d = np.hypot(x, yi)
z = 10 * np.cos(d) / (d+1)
pts = np.column_stack([x, np.full_like(x, yi), z])
```

On the other hand, with the use of hypot, it's even less obvious what function is being plotted.

Reply:
Your suggestion performed better, so I'll incorporate it, given this is an example and its functional, I see no reason not to make a change.